documentation/docs/how_tos/cli/running_examples.mdx

```diff
@@ -0,0 +1,23 @@
+---
+description: History behind the development of the Stronghold Engine library that reflects upon previous revisions and the lessons learnt.
```

Comment:
Hm is this description taken from somewhere else? I would say it fits partially.

Reply:
```suggestion
You can run Stronghold examples using the command line interface from within the client crate.
```